.github/workflows/ci.yaml

```diff
@@ -51,5 +51,18 @@ jobs:
         profile: minimal
         toolchain: nightly
         components: rustfmt
-    - run: cargo new project && cd project && cargo build
-    # todo, add that it actually used it, maybe by show stats
\ No newline at end of file
+    - name: Create and build test project
+      run: |
+        cargo new project --lib
+        cd project
+        cargo build
+      env:
+        CARGO_INCREMENTAL: 0
```

Comment:
Since we set `RUSTC_WRAPPER` as part of the action, should we be setting `CARGO_INCREMENTAL` as well?

Reply:
I don't think so, atleast not by default.

Reply:
Nice catch! I think if it's required for scache to work that makes sense to set

Reply:
Ok! Will add that in a separate PR

Reply:
Ok! I'll add that in a separate PR

Reply:
Ok! I'll add that in a separate PR

Reply:
Opened https://github.com/metalbear-co/sccache-action/pull/9